<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>converter</class>
 <widget class="QMainWindow" name="converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>converter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="BT_Choose_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_Choose_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>230</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Choose</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_Quit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>(Q)uit</string>
    </property>
    <property name="shortcut">
     <string>Esc, Esc, Ctrl+Q</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_App">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>450</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>App</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_Exec">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>450</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Execute</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Input_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="LB_Input_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Input 2</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_Input_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="TE_Input_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionApp"/>
    <addaction name="actionExecute"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionApp">
   <property name="text">
    <string>App</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
